terraform/projects/app-ecs-services/task-definitions/prometheus-server.json

```diff
@@ -19,6 +19,10 @@
       {
         "sourceVolume": "alert-config",
         "containerPath": "/etc/prometheus/alerts"
+      },
+      {
+        "sourceVolume": "prometheus-timeseries-storage",
+        "containerPath": "/prometheus"
```

Comment:
can we document that this is default directory that our docker image expects? i was confused for a while because i couldn't see how this got wired up

Reply:
👍